README.md

```diff
@@ -1,3 +1,6 @@
+![GirlScript Season of Code](https://github.com/pankaj-bind/GameZone/blob/main/assets/images/GSSoC.png)
+
+
 <div style="display:none;" align="center"><h1>
```

Comment:
@pankaj-bind there are certain changes in README that we are configuring right now !